### ISSUE TITLE
RaisedButton gives exception as StatelessElement.build

### ISSUE THREAD

**Issue by pawanadubey** [flutter/flutter#23460]
Hi all,

I'm trying to add a RaisedButton in the container, however I'm getting below exception.
```
[{"event":"app.progress","params":{"appId":"9d38bcd5-7d0a-4cfe-a121-20c0dff4c237","id":"10","progressId":"hot.reload","message":"Performing hot reload..."}}]Performing hot reload...

Syncing files to device Android SDK built for x86...
I/flutter ( 5093): ══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
I/flutter ( 5093): The following _CompileTimeError was thrown building MyApp(dirty):
I/flutter ( 5093): 'file:///D:/LearningMaterials/Flutter/udemy_learningapp/lib/main.dart': error: Unexpected tag 128
I/flutter ( 5093): (SpecializedVariableGet) in ?, expected a procedure, a constructor or a function node
I/flutter ( 5093): When the exception was thrown, this was the stack:
I/flutter ( 5093): #0      StatelessElement.build (package:flutter/src/widgets/framework.dart:3731:28)
I/flutter ( 5093): #1      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3678:15)
I/flutter ( 5093): #2      Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
I/flutter ( 5093): #3      BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2273:33)
I/flutter ( 5093): #4      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding&WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:673:20)
I/flutter ( 5093): #5      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:208:5)
I/flutter ( 5093): #6      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:990:15)
I/flutter ( 5093): #7      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:930:9)
I/flutter ( 5093): #8      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.scheduleWarmUpFrame.<anonymous closure> (package:flutter/src/scheduler/binding.dart:751:7)
I/flutter ( 5093): #10     _Timer._runTimers (dart:isolate/runtime/libtimer_impl.dart:382:19)
I/flutter ( 5093): #11     _Timer._handleMessage (dart:isolate/runtime/libtimer_impl.dart:416:5)
I/flutter ( 5093): #12     _RawReceivePortImpl._handleMessage (dart:isolate/runtime/libisolate_patch.dart:171:12)
I/flutter ( 5093): (elided one frame from package dart:async)
I/flutter ( 5093): ════════════════════════════════════════════════════════════════════════════════════════════════════
Reloaded 1 of 412 libraries in 969ms.
```

**Comment by zoechi** [comment]
Please add the output of `flutter doctor -v`
and the output of `flutter analyze`

**Comment by pawanadubey** [comment]
flutter doctor -v output:
[ΓêÜ] Flutter (Channel beta, v0.9.4, on Microsoft Windows [Version 6.1.7601], lo
cale en-US)
    ΓÇó Flutter version 0.9.4 at C:\flutter
    ΓÇó Framework revision f37c235c32 (4 weeks ago), 2018-09-25 17:45:40 -0400
    ΓÇó Engine revision 74625aed32
    ΓÇó Dart version 2.1.0-dev.5.0.flutter-a2eb050044

[ΓêÜ] Android toolchain - develop for Android devices (Android SDK 28.0.2)
    ΓÇó Android SDK at C:\Users\pawadube\AppData\Local\Android\sdk
    ΓÇó Android NDK location not configured (optional; useful for native profili
ng support)
    ΓÇó Platform android-28, build-tools 28.0.2
    ΓÇó Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    ΓÇó Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b
06)
    ΓÇó All Android licenses accepted.

[ΓêÜ] Android Studio (version 3.2)
    ΓÇó Android Studio at C:\Program Files\Android\Android Studio
    ΓÇó Flutter plugin version 29.1.1
    ΓÇó Dart plugin version 181.5656
    ΓÇó Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b
06)

[ΓêÜ] Connected devices (1 available)
    ΓÇó Android SDK built for x86 ΓÇó emulator-5554 ΓÇó android-x86 ΓÇó Android
6.0 (API 23) (emulator)

ΓÇó No issues found!


flutter analyze output is attached in the file.
[analyze.txt](https://github.com/flutter/flutter/files/2510447/analyze.txt)

**Comment by zoechi** [comment]
What directory did you run `flutter analyze` in?
It reports tons about unrelated code.
Can you please run it again in your project directory (the directory that contains `pubspec.yaml`)

**Comment by zoechi** [comment]
Please also try `flutter clean` and then run again to see if the problem persists.

**Comment by pawanadubey** [comment]
I tried flutter clean, but no luck.
flutter analyze output
Analyzing udemy_learningapp...

   info - Unused import: 'package:udemy_learningapp/main.dart' - test\widget_tes
t.dart:10:8 - unused_import
  error - The constructor returns type 'dynamic' that isn't of expected type 'Wi
dget' - test\widget_test.dart:15:29 - strong_mode_invalid_cast_new_expr
  error - Undefined class 'MyApp' - test\widget_test.dart:15:33 - undefined_clas
s

3 issues found. (ran in 5.2s)

If I replace the RaisedButton by Text widget it works fine. Below is the code:
class _MyStateFulWidgetState extends State<MyStateFulWidget> {
  @override
  Widget build(BuildContext context) {
    return Container(
        color: Colors.green,
        child: new RaisedButton(
          onPressed: null,
          child: new Text(
            "Click",
            textDirection: TextDirection.ltr,
          ),
        ));
  }
}

**Comment by zoechi** [comment]
That's really weird.
I'd suggest in the Flutter installation directory:
```
git clean -xfd
git pull
flutter doctor
```

**Comment by pawanadubey** [comment]
I ran the git commands, and then I tried flutter doctor now I'm getting below error:

C:\flutter\bin>flutter doctor
Checking Dart SDK version...
The term 'Unblock-File' is not recognized as the name of a cmdlet, function, sc
ript file, or operable program. Check the spelling of the name, or if a path wa
s included, verify that the path is correct and try again.
At line:1 char:13
+ Unblock-File <<<<  -Path 'C:\flutter/bin/internal/update_dart_sdk.ps1'; & 'C:
\flutter/bin/internal/update_dart_sdk.ps1'
    + CategoryInfo          : ObjectNotFound: (Unblock-File:String) [], Comman
   dNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Downloading Dart SDK from Flutter engine 74625aed323d04f2add0410e84038d250f51b61
6...
Downloading the Dart SDK using the BITS service failed, retrying with WebRequest
...
The term 'Invoke-WebRequest' is not recognized as the name of a cmdlet, functio
n, script file, or operable program. Check the spelling of the name, or if a pa
th was included, verify that the path is correct and try again.
At C:\flutter\bin\internal\update_dart_sdk.ps1:53 char:22
+     Invoke-WebRequest <<<<  -Uri $dartSdkUrl -OutFile $dartSdkZip
    + CategoryInfo          : ObjectNotFound: (Invoke-WebRequest:String) [], P
   arentContainsErrorRecordException
    + FullyQualifiedErrorId : CommandNotFoundException

Error: Unable to update Dart SDK. Retrying...

Waiting for 0 seconds, press CTRL+C to quit ...
Checking Dart SDK version...
The term 'Unblock-File' is not recognized as the name of a cmdlet, function, sc
ript file, or operable program. Check the spelling of the name, or if a path wa
s included, verify that the path is correct and try again.
At line:1 char:13
+ Unblock-File <<<<  -Path 'C:\flutter/bin/internal/update_dart_sdk.ps1'; & 'C:
\flutter/bin/internal/update_dart_sdk.ps1'
    + CategoryInfo          : ObjectNotFound: (Unblock-File:String) [], Comman
   dNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Can you please check what went wrong?

**Comment by zoechi** [comment]
Looks like you need to update PowerShell https://stackoverflow.com/questions/29933822/powershell-to-unblock-all-the-files-in-a-folder

**Comment by zoechi** [comment]
@qurm you need to pass a value to `Text(...)` that is **not** `null`

**Comment by qurm** [comment]
Thanks, and I agree, after further investigation.  Was just coincidental with a Flutter upgrade, so was checking the issue list...

**Comment by zoechi** [comment]
Duplicate of #17204
